loghub/main.py

```diff
@@ -164,7 +166,7 @@ def format_changelog(repo,
         version=version, date=close_date, q=quotes)
 
     lines.append(header)
-    lines.append('### Bugs fixed\n')
+    lines.append('### Issues closed\n')
```

Comment:
Then lets use only

Issues Closed

Pull Requests Closed (And we can filter by the ones that were merged also?)